release/ARTIFACT_BUCKET

```diff
@@ -0,0 +1 @@
+prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp
```

Comment:
Yea I feel that since this is not something we would actively change like RELEASE or GIT_TAG, wondering if we can store in a common constants file or something. Also would rather do what is considered "best practice" with regards to individual files vs a constants file.